backend/src/main/java/gov/cdc/usds/simplereport/db/model/PersonUtils.java

```diff
@@ -829,4 +829,25 @@ public static final Map<String, String> getResidenceTypeMap() {
           TRANS_WOMAN, "Female gender identity",
           OTHER, "Non-binary gender identity",
           REFUSED, "Asked But Declined");
+
+  public static Map<String, String> getGenderIdentityAbbreviationMap() {
```

Comment:
nit: could keep the existing style for instantiating the maps here consistent and use `genderIdentityAbbrevationMap = Map.of(...)`?

Reply:
IIRC using `Map.of(...)` in that way only works for up to 10 elements :(

Reply:
Think we might be able to use `Map.ofEntries` in that case! 

Reading up on how that method is implemented under the hood made my jaw drop. 
![Screenshot 2024-03-26 at 9 50 39 AM](https://github.com/CDCgov/prime-simplereport/assets/29645040/0bd8b78b-325e-4a07-87b2-2dc4d7f69077)